lib/arborist/build-ideal-tree.js

```diff
@@ -130,6 +135,12 @@ module.exports = cls => class IdealTreeBuilder extends cls {
     this[_linkNodes] = new Set()
     this[_manifests] = new Map()
     this[_peerConflict] = null
+
+    // caches for cached realpath calls
+    const cwd = process.cwd()
+    // assume that the cwd is real enough for our purposes
+    this[_rpcache] = this[_rpcache] || new Map([[ cwd, cwd ]])
+    this[_stcache] = this[_stcache] || new Map()
```

Comment:
just delete these two lines, no need to re-initialize since ActualLoader does it.